docs/auth/gitlab/provider.md

```diff
@@ -18,7 +18,7 @@ Settings for local development:
 
 - Name: Backstage (or your custom app name)
 - Redirect URI: `http://localhost:7007/api/auth/gitlab/handler/frame`
-- Scopes: `read_api` and `read_user`
+- Scopes: `openid` and `read_user`
```

Comment:
How come `read_api` was removed?

Reply:
dunno I think this was out-of-date tbh -- `GitlabAuth` was only requesting `read_user`